go-tools/Makefile

```diff
@@ -7,7 +7,9 @@ SUFFIX = $(GOOS)-$(GOARCH)
 UNVERSIONED_TOOLS = golang.org/x/exp/cmd/apidiff \
 					github.com/robertkrimen/godocdown/godocdown \
 					golang.org/x/vuln/cmd/govulncheck \
-					github.com/kisielk/godepgraph
+					github.com/kisielk/godepgraph \
+					github.com/subfuzion/envtpl/cmd/envtpl \
```

Comment:
envtpl 1.0.0 is the only version tagged, and is severely (>4 years) out of date, so effectively yes. This should probably pull from `@master` instead of `@latest`.

Reply:
Hesitantly changed the Makefile to fetch `envtpl` from `master`, since the single tag misleads `@latest`. Let me know if that change isn't appropriate

Reply:
Seems reasonable.